tests/reviewer_tools/test_reviewer_tools_release.py

```diff
@@ -239,3 +259,15 @@ def test_queues_ratings_awaiting_moderation_tc_id_C4586(selenium, base_url):
     d) A "Process reviews" blue button (at the beginning and on the end of the reviews table)"""
     ratings_awaiting_moderation.assert_moderation_actions_section_elements()
     ratings_awaiting_moderation.assert_process_reviews_buttons()
+
+@pytest.mark.login("reviewer_user")
+def test_content_approve_version_tc_id_T5456486(selenium, base_url):
+    """Submit a new add-on in order to test the content approve function"""
+    addon_slug = submit_addon_method(selenium, base_url)
+    """Load AMO Reviewer Tools homepage"""
+    reviewer_tools_homepage = ReviewerToolsHomepage(selenium, base_url).open().wait_for_page_to_load()
```

Comment:
If it helps you could just use the `#` symbol. Usually the 6 marks like `"""text"""` are used for bigger blocks of text that span multiple lines.

```suggestion
    # Load AMO Reviewer Tools homepage
    reviewer_tools_homepage = ReviewerToolsHomepage(selenium, base_url).open().wait_for_page_to_load()
```

Reply:
Hello, thank you for the suggestion. I updated the single lines of text